plugins/omemo/src/stream_module.vala

```diff
@@ -58,6 +60,10 @@ public class StreamModule : XmppStreamModule {
                     .put_node(new StanzaNode.build("payload", NS_URI)
                         .put_node(new StanzaNode.text(Base64.encode(ciphertext))));
 
+            StanzaNode encryption = new StanzaNode.build("encryption", NS_URI_0380).add_self_xmlns()
```

Comment:
I guess in theory one could write a function `encryption_stanza` that returns:

```vala
return new StanzaNode.build("encryption", NS_URI_0380)
                     .add_self_xmlns()
                     .put_attribute("namespace", NS_URI)
```

but the code that is duplicated here does not look to bad here.